pytorch_lightning/root_module/root_module.py

```diff
@@ -113,6 +113,14 @@ def optimizer_step(self, epoch_nb, batch_nb, optimizer, optimizer_i, second_orde
         # clear gradients
         optimizer.zero_grad()
 
+    def tbptt_split_batch(self, batch):
```

Comment:
@tullie is there a default implementation we can add here that would apply to 90% of use cases?

Something like:

```python
def split(batch):
   for i, x in enumerate(batch):
        # maybe recurse if we think the inputs can be lists of lists or of dicts
        if isinstance(x, list):
             x = split(x)
        
        batch[i] = some_split_fx(x)

   return batch
```

Reply:
Yeah, if we pass in the tbptt step size in the parameter then writing the default will be easier. I'll go ahead with that.